pyvrp/cpp/crossover/selective_route_exchange.cpp

```diff
@@ -12,18 +12,22 @@ Individual selectiveRouteExchange(
     std::pair<Individual const *, Individual const *> const &parents,
     ProblemData const &data,
     PenaltyManager const &penaltyManager,
-    XorShift128 &rng)
+    size_t startA,
+    size_t startB,
+    size_t const nMovedRoutes)
 {
     size_t nRoutesA = parents.first->numRoutes();
     size_t nRoutesB = parents.second->numRoutes();
 
-    // Picking the start index of routes to replace of parent A. We like to
-    // replace routes with a large overlap of tasks, so we choose adjacent
-    // routes (they are sorted on polar angle by the local search/educate step).
```

Comment:
I hadn't looked at this method in a long long time, but now that I do: this is really dependent on a super non obvious detail of the current LS implementation. I don't think we need to change that in this PR, but can we keep the comment that explains we rely on that polar angle sort somehow?